kele.el

```diff
@@ -1389,6 +1412,55 @@ Defaults to the currently active context as set in
   :options (lambda ()
              (alist-get 'group-versions (oref transient--prefix scope))))
 
+(transient-define-suffix kele-list (group-version kind context namespace)
+  "List all resources of a given GROUP-VERSION and KIND.
+
+If CONTEXT is provided, use it.  Otherwise, use the current context as reported
+by `kele-current-context-name'.
+
+If NAMESPACE is provided, use it.  Otherwise, use the default
+namespace for the context.  If NAMESPACE is provided and the KIND
+is not namespaced, returns an error."
+  :key "l"
+  :description
+  (lambda ()
+    (format "List all %s"
+            (propertize
+             (alist-get 'kind (oref transient--prefix scope))
+             'face
+             'warning)))
+  (interactive
+   (let* ((kind (alist-get 'kind (oref transient-current-prefix scope)))
+          (args (transient-args transient-current-command))
+          (group-version (transient-arg-value "--groupversion=" args))
+          (namespace (transient-arg-value "--namespace=" args))
+          (context (transient-arg-value "--context=" args)))
+     (list group-version kind context namespace)))
```

Comment:
This should be implemented in such a way that a user can do `M-x kele-list` and still have it be usable. That is not the case currently.